src/features/fwiCalculator/components/TimeRangeSlider.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import IOSSlider from 'features/fwiCalculator/components/IOSSlider'
+import { InputLabel, makeStyles } from '@material-ui/core'
+
+const useStyles = makeStyles({
+  root: {
+    marginTop: 20
+  },
+  inputLabel: {
+    marginBottom: 5
+  }
+})
+
+interface Props {
+  timeRange: number
+  onYearRangeChange: (yearRangeNumber: number) => void
+}
+
+const MIN_YEARS = 0
+
+const MAX_YEARS = 50
```

Comment:
```const MAX_YEARS = 50```
will stop working next year. The earliest date record in the database is for 1970. So 2020-50 works right now. But next year it will be 2021-50=1971, and we'll start missing data.

Reply:
Now dynamically assigning the value (currYear - 1970).